### ISSUE TITLE
Contribute: Mention ready-to-test Pull requests

### ISSUE THREAD

**Issue by toby63** [mumble-voip/mumble-www#137]
For example @TerryGeng is searching for testers for this PR (https://github.com/mumble-voip/mumble/pull/4694), so it might be useful to:
- add a special label (in main repo), like "test-requested" or similar
- mention this label in the contribute guide

What do you think?

**Comment by Krzmbrzl** [comment]
There already is such a label: `needs-testing`

**Comment by TerryGeng** [comment]
Thanks for the follow-up.

The PR you have mentioned is still considered to be unmatured and more work has to be done in order to get it merged. I will change it into WIP.

**Comment by toby63** [comment]
@TerryGeng 
> The PR you have mentioned is still considered to be unmatured and more work has to be done in order to get it merged. I will change it into WIP.

Ok, so you don't want testers for now.

---

@Krzmbrzl While that is good, it doesn't really answer the question, should I add a mention of that or not?
There are multiple PRs we had, that would fit, mostly audio, but also grpc etc.
Even though right now, not so much is going on.

**Comment by TerryGeng** [comment]
> Ok, so you don't want testers for now.

I do want someone to tell me if it works and in what situation it works or doesn't work.

Can you help me test it?

**Comment by Krzmbrzl** [comment]
> should I add a mention of that or not?

I don't think that it would change that much since we can't label a PR and then leave it there until someone external finally comes around testing it. Thus I expect PRs having these labels only for a rather short period of time.
I think people should rather focus on testing development snapshots and/or build from current master and test that. That would probably fit our development process better :thinking:

**Comment by toby63** [comment]
@TerryGeng 
> I do want someone to tell me if it works and in what situation it works or doesn't work.

That's the definition of Testing.

> Can you help me test it?

No, I don't have a Mac.

---

> I think people should rather focus on testing development snapshots and/or build from current master and test that. That would probably fit our development process better :thinking:

Well it's all your decision.
One argument for the mention is that testing was requested multiple times before the PRs are merged.
If we mention it, more people could be aware of that.
